server/src/main/java/com/scalar/db/server/DistributedTransactionService.java

```diff
@@ -473,6 +473,7 @@ private void execute(
                 .setMessage(e.getMessage())
                 .build());
       } catch (UnknownTransactionStatusException e) {
+        logger.error("the transaction status is unknown", e);
```

Comment:
If `UnknowTransactionStatusException.getUnknownTransactionId()` is present, should we append the transaction id to the log message, or it is redundant?

Reply:
Thank you for pointing it out. Yes, we should show the transaction ID as well. I'll fix it. Thanks.

Reply:
@Torch3333 I just fixed it. Please take a look again when you have time!